pyproject.toml

```diff
@@ -43,20 +42,20 @@ dependencies = [
   "defusedcsv>=2.0,<3.0",
   "defusedxml>=0.7.1,<1.0",
   "django>=4.2,<5.0",
-  "django-cleanup>=8.0,<9.0",
+  "django-cleanup>=8.0,<10.0",
   "django-compressor>=4.4,<5.0",
   "django-extensions>=3.2,<4.0",
   "django-filter>=23.2,<25.0",
   "django-libsass>=0.9,<1.0",
   "django-mathfilters>=1.0,<2.0",
-  "django-mptt==0.14.0",  # pinned, 0.15 requires Python >= 3.9
+  "django-mptt>=0.16.0,<1.0.0",
   "django-rest-swagger>=2.2,<3.0",
   "django-settings-export>=1.2,<2.0",
   "django-split-settings>=1.2,<2.0",
   "django-widget-tweaks>=1.5,<2.0",
   "djangorestframework>=3.15,<4.0",
   "drf-extensions>=0.7.1,<1.0",
-  "importlib_metadata<8.5.0",  # upstrem problem, see https://github.com/python/importlib_metadata/issues/506
+  "importlib_metadata<8.7.0",  # upstrem problem, see https://github.com/python/importlib_metadata/issues/506
```

Comment:
Do we still need this pin? I added it, but I thought maybe they will fix it.

Reply:
ok, now I see, this package was not even in the dependencies before it needed to be pinned. Think I'll remove it then.

Reply:
I've removed it now